dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/AbstractConfig.java

```diff
@@ -590,6 +590,15 @@ public String toString() {
                     if (MethodUtils.isGetter(method)) {
                         String name = method.getName();
                         String key = calculateAttributeFromGetter(name);
+                        /**
+                         * make sure attribute exists
+                         */
+                        try {
+                            getClass().getField(key);
```

Comment:
Some attribute does not has a corresponding field, but has a pair of getter/setter methods, e.g. String getInterface()/setInterface(String) of ReferenceConfig. 
So it’s better to check if the attribute is writable, see aslo: https://github.com/apache/dubbo/pull/8289/